aptos-move/framework/aptos-stdlib/sources/cryptography/bls12381.move

```diff
@@ -330,369 +400,123 @@ module aptos_std::bls12381 {
         message: vector<u8>
     ): bool;
 
-    //
-    // Constants and helpers for tests
-    //
+    #[test_only]
+    /// Generates a bls12381 key pair.
+    native fun generate_keys_internal(): (vector<u8>, vector<u8>);
 
-    /// Random signature generated by running `cargo test -- bls12381_sample_signature --nocapture --include-ignored` in `crates/aptos-crypto`.
-    /// The message signed is "Hello Aptos!" and the associated SK is 07416693b6b32c84abe45578728e2379f525729e5b94762435a31e65ecc728da.
-    const RANDOM_SIGNATURE: vector<u8> = x"a01a65854f987d3434149b7f08f70730e30b241984e8712bc2aca885d632aafced4c3f661209debb6b1c8601326623cc16ca2f6c9edc53b7b88b7435fb6b05ddece418d2c34dc6aca2f5a11a79e67774582c14084a01dcb7820e4cb4bad0ea8d";
+    #[test_only]
+    /// Generates a signature for a message with a signing key.
+    native fun sign_internal(sk: vector<u8>, msg: vector<u8>): vector<u8>;
+
+    #[test_only]
+    /// Generates a proof-of-possession from a secret key.
```

Comment:
/// Generates a proof-of-possession (PoP) for the public key associated with the secret key `sk`.